docs/docs-content/getting-started/aws/scale-secure-cluster.md

```diff
@@ -0,0 +1,505 @@
+**Projects** page.
+
+Navigate to the left **Main Menu** and click on **Users & Teams**.
+
+Select the **Teams** tab. Then, click on **Create Team**.
+
+Fill in the **Team Name** with **scale-secure-tutorial-team**. Click on **Confirm**.
+
+Once Palette creates the team, select it from the **Teams** list. The **Team Details** pane opens.
+
+On the **Project Roles** tab, click on **New Project Role**. The list of project roles appears.
+
+Select the **Project-ScaleSecureTutorial** from the **Projects** drop-down. Then, select the **Cluster Profile Viewer**
+and **Cluster Viewer** roles. Click on **Confirm**.
+
+![Image that shows how to select team roles](/getting-started/getting-started_scale-secure-cluster_select-team-roles.webp)
+
+Any users that you add to this team inherit the project roles assigned to it. Roles are the foundation of Palette's RBAC
+enforcement. They allow a single user to have different types of access control based on the resource being accessed. In
+this scenario, any user added to this team will have access to view any cluster profiles and clusters in the
+**Project-ScaleSecureTutorial** project, but not modify them. Check out the
+[Palette RBAC](../../user-management/palette-rbac/palette-rbac.md) section for more details.
+
+Navigate to the left **Main Menu** and click on **Projects**.
+
+Click on **Open project** on the **Project-ScaleSecureTutorial** card.
+
+![Image that shows how to open the tutorial project](/getting-started/getting-started_scale-secure-cluster_open-tutorial-project.webp)
+
+Your scope changes from **Tenant Admin** to **Project-ScaleSecureTutorial**. All further resources you create will be
+part of this project.
+
+## Import a Cluster Profile
+
+Palette provides three resource contexts. They help you customize your environment to your organizational needs, as well
+as control the scope of your settings.
+
+| Context | Description                                                                              |
+| ------- | ---------------------------------------------------------------------------------------- |
+| System  | Resources are available at the system level and to all tenants in the system.            |
+| Tenant  | Resources are available at the tenant level and to all projects belonging to the tenant. |
+| Project | Resources are available within a project and not available to other projects.            |
+
+All of the resources you have created as part of your Getting Started journey have used the **Project** context. They
+are only visible in the **Default** project. Therefore, you will need to create a new cluster profile in
+**Project-ScaleSecureTutorial**.
+
+Navigate to the left **Main Menu** and click on **Profiles**. Click on **Import Cluster Profile**. The **Import Cluster
+Profile** pane opens.
+
+Paste the following in the text editor. Click on **Validate**. The **Select repositories** dialog appears.
+
+**TODO: Replace when the pack is published!**
```

Comment:
Is the pack published? :stuck_out_tongue_winking_eye:

Reply:
Yes! Updated now.